Mathlib/Tactic/Linter/CasesLinter.lean

```diff
@@ -0,0 +1,55 @@
+/-
+Copyright (c) 2025 Jeremy Tan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jeremy Tan, Damiano Testa
+-/
+import Lean.Elab.Command
+-- Import this linter explicitly to ensure that
```

Comment:
I'm partial to removing this comment also: this is enforced by a mathlib linter anyway.